bungee/src/main/java/codecrafter47/bungeetablistplus/handler/AbstractLegacyTabOverlayHandler.java

```diff
@@ -84,6 +86,7 @@ public abstract class AbstractLegacyTabOverlayHandler implements PacketHandler,
 
     private static final String[][] EMPTY_PROPERTIES = new String[0][];
     private static final String EMPTY_JSON_TEXT = "{\"text\":\"\"}";
+    private static final TextComponent EMPTY_COMPONENT_TEXT = new TextComponent();
```

Comment:
Just a nitpick, but the 3 files that changed have three different naming schemes for these 2 properties

Reply:
good catch!